tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/SdkRepoConfigHelper.cs

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ISdkRepoConfigHelper
+    {
+        // Config value retrieval methods
+        Task<T> GetConfigValueFromRepoAsync<T>(string repositoryRoot, string jsonPath);
+        Task<(BuildConfigType type, string value)> GetBuildConfigurationAsync(string repositoryRoot);
+        
+        // Command processing methods
+        string SubstituteCommandVariables(string command, Dictionary<string, string> variables);
+        string[] ParseCommand(string command);
+    }
+
+    public class SdkRepoConfigHelper : ISdkRepoConfigHelper
```

Comment:
This name might be too generic. Just looking at it with no context I would have no idea what it's supposed to do or what config it references.

Reply:
Renamed to `SpecGenSdkConfigHelper` and added comment to provide context.